api/server.go

```diff
@@ -628,6 +629,31 @@ func (s *server) pubkeyHostHandler(jc jape.Context) {
 	jc.Encode(hosts[0])
 }
 
+func (s *server) hostsListHandler(jc jape.Context) {
+	var params explorer.HostQuery
+	if jc.Decode(&params) != nil {
+		return
+	}
+
+	limit := uint64(100)
```

Comment:
Just wanted to briefly draw attention to the default limit here. In `renterd` we default to unlimited (-1), in `hostd` we default to 100 but apply a 500 max limit. Would be nice to get these small things consistent across our APIs.

Reply:
Might be fine for the explorer to be a bit stricter since unknown third parties send these requests where in other places we know only trusted parties have access.